m/main_glider_data_processing_dt.m

```diff
@@ -426,54 +467,63 @@
-    end
-  end
+      disp(['Files loaded in deployment period: ' num2str(numel(source_files)) '.']);
+      deployment.source_files = sprintf('%s\n', source_files{:});
+  end  
   
+% $$$   %% Generate L0 NetCDF file (raw/preprocessed data), if needed and possible.
+% $$$   if ~isempty(fieldnames(data_raw)) && ~isempty(netcdf_l0_file)
+% $$$     disp('Generating NetCDF L0 output...');
+% $$$     try
+% $$$       switch glider_type
+% $$$         case {'slocum_g1' 'slocum_g2'}
+% $$$           outputs.netcdf_l0 = generateOutputNetCDF( ...
+% $$$             netcdf_l0_file, data_raw, meta_raw, deployment, ...
+% $$$             netcdf_l0_options.variables, ...
+% $$$             netcdf_l0_options.dimensions, ...
+% $$$             netcdf_l0_options.attributes, ...
+% $$$             'time', {'m_present_time' 'sci_m_present_time'}, ...
+% $$$             'position', {'m_gps_lon' 'm_gps_lat'; 'm_lon' 'm_lat'}, ...
+% $$$             'position_conversion', @nmea2deg, ...
+% $$$             'vertical',            {'m_depth' 'sci_water_pressure'}, ...
+% $$$             'vertical_conversion', {[]        @(z)(z * 10)}, ...
+% $$$             'vertical_positive',   {'down'} );
+% $$$         case 'seaglider'
+% $$$           outputs.netcdf_l0 = generateOutputNetCDF( ...
+% $$$             netcdf_l0_file, data_raw, meta_raw, deployment, ...
+% $$$             netcdf_l0_options.variables, ...
+% $$$             netcdf_l0_options.dimensions, ...
+% $$$             netcdf_l0_options.attributes, ...
+% $$$             'time', {'elaps_t'}, ...
+% $$$             'time_conversion', @(t)(t + meta_raw.start_secs), ... 
+% $$$             'position', {'GPSFIX_fixlon' 'GPSFIX_fixlat'}, ...
+% $$$             'position_conversion', @nmea2deg, ...
+% $$$             'vertical',            {'depth'}, ...
+% $$$             'vertical_conversion', {@(z)(z * 10)}, ... 
+% $$$             'vertical_positive',   {'down'} );
+% $$$         case {'seaexplorer'}
+% $$$           outputs.netcdf_l0 = generateOutputNetCDF( ...
+% $$$               netcdf_l0_file, data_raw, meta_raw, deployment, ...
+% $$$               netcdf_l0_options.variables, ...
+% $$$               netcdf_l0_options.dimensions, ...
+% $$$               netcdf_l0_options.attributes, ...
+% $$$               'time', {'Posixtime_nav' 'Posixtime_sci'}, ...
+% $$$               'position', {'NAV_LONGITUDE' 'NAV_LATITUDE'; 'Lon' 'Lat'}, ...
+% $$$               'position_conversion', @nmea2deg, ...
+% $$$               'vertical',            {'Depth' 'SBD_PRESSURE'}, ...
+% $$$               'vertical_conversion', {[]        @(z)(z * 10)}, ...
+% $$$               'vertical_positive',   {'down'} );      
+% $$$       end
+% $$$       disp(['Output NetCDF L0 (raw data) generated: ' outputs.netcdf_l0 '.']);
+% $$$     catch exception
+% $$$       disp(['Error generating NetCDF L0 (raw data) output ' netcdf_l0_file ':']);
+% $$$       disp(getReport(exception, 'extended'));
+% $$$     end
+% $$$   end
```

Comment:
Just to note that you can disable the generation of the L0 file by setting its path to empty in `configDTPathsLocal`.
It is ok if you want to leave it here, however, the original code will appear again after merging.